examples/js-cloud-server/cloudflare-worker/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "js-cloud-server-sdk-test-worker",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "deploy": "wrangler deploy",
+    "start": "wrangler dev"
+  },
+  "dependencies": {
+    "@devcycle/js-cloud-server-sdk": "1.0.0"
```

Comment:
examples arent in the workspace anymore are they?

Reply:
they are still in the workspace: 
<img width="975" alt="image" src="https://github.com/DevCycleHQ/js-sdks/assets/1219069/378e5c79-9cbb-4247-8f93-626e1c8de4e5">

Reply:
this is a bit weird because its gonna basically symlink the cloud server SDK into this directory's node_modules but then (hopefully) ignore it during builds due to the tsconfig alias